docs/20_tutorial.md

```diff
@@ -2,3 +2,63 @@
 
 Tutorial
 ========
+
+***should the tutorial deal with the SOS establishing the risk
+limit, or any of their pre-audit responsibilities like the selection
+of which races to audit?
+
+Brief introduction to RLAs in non-technical, human friendly language.
+At the county level, something along the lines of "This tool is going
+to help you cross reference votes recorded on paper ballots with how 
+the computers read those ballots, to determine whether the computers
```

Comment:
Agreed.  But I would suggest substituting "system" for "computer", since the CVR represents the end result of a complex interaction of equipment, computers, and humans doing manual adjudication on some questionable ballots. Perhaps:
> How the system interpreted those ballots...
> to determine whether the system functioned properly